configs/projects/_shared-agent-components.rb

```diff
@@ -69,3 +69,6 @@
 if platform.is_macos?
   proj.component 'rubygem-CFPropertyList'
 end
+
+proj.component 'rubygem-strscan'
```

Comment:
`rexml 3.0.7` does require `strscan >= 3.0.9`, but there was a large backlash from the community, because `strscan 3.x` contains native extensions, which meant you needed a compiler to install `rexml` and there were JRuby incompatibilities.

The `strscan` requirement was later relaxed, so [`rexml` 3.2.9](https://rubygems.org/gems/rexml/versions/3.2.9) now accepts any version of `strscan`.  See https://github.com/ruby/rexml/commit/f1df7d13b3e57a5e059273d2f0870163c08d7420#diff-75d0a867bbfd5e375164c17cf7a9df96761fb050e7308f3918ad6fe1fe12d27d 

In other words, if we bump to `rexml 3.2.9`, then we can drop the `rubygem-strscan` component, since it's a default gem https://stdgems.org/2.7.8/#default-gems.

Reply:
Thanks @joshcooper,
Yes, I have updated the files in this PR bumping to 3.2.9.
I am currently testing some platforms for sanity with that on my local. I will fully update this PR in a while.